### PR TITLE
solution14 completed

### PR DESCRIPTION
**Pull request by Jayshgandhi**
Given complex numbers:
15.8  18.9i
9.8  6.9i

Addition result:
ComplexNumber{real=25.6, imaginary=25.8i }

Subtraction result:
ComplexNumber{real=6.0, imaginary=12.0i }

Division result:
ComplexNumber{real=154.84001, imaginary=130.41i }

### CODE REVIEW

File: Exercises/Excercise14/solution14.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+        }
+        ComplexNumber complexSub(ComplexNumber c,ComplexNumber f){
+            System.out.println();
+            System.out.println("Subtraction result:");
+            return new ComplexNumber (super.sub(c.real,f.real),super.sub(c.imaginary, f.imaginary));
+        }
+        ComplexNumber complexMul(ComplexNumber c,ComplexNumber f){
+            System.out.println();
+            System.out.println("Division result:");
+            return new ComplexNumber (super.mul(c.real,f.real),super.mul(c.imaginary, f.imaginary));
```
Comment (hockeyjudson) [inline Exercises/Excercise14/solution14.java]: Check the logic for complex number multiplication

**Review by hockeyjudson — CHANGES_REQUESTED** [review]
Please change it

File: Exercises/Excercise14/solution14.java (hunk 1 above)
Comment (Jayshgandhi) [reply Exercises/Excercise14/solution14.java]: Modified the logic for complex multiplication:
Given complex numbers:
15.8  18.9i
9.8  6.9i

Addition result:
ComplexNumber{real=25.6, imaginary=25.8i }

Subtraction result:
ComplexNumber{real=6.0, imaginary=12.0i }

Multiplication result:
ComplexNumber{real=24.430008, imaginary=294.24i }

**Review by hockeyjudson — APPROVED** [review]
LGTM

**Pull request merged**